openslides_backend/action/actions/motion_submitter/sort.py

```diff
@@ -27,3 +32,12 @@ def get_updated_instances(self, action_data: ActionData) -> ActionData:
             filter_id=instance["motion_id"],
             filter_str="motion_id",
         )
+
+    def get_meeting_id(self, instance: Dict[str, Any]) -> int:
```

Comment:
Doesn't the submitter have a `meeting_id`?

Reply:
A submitter has a meeting_id, what is the point?

Reply:
nevermind